### PR TITLE
Create docker-image.yml

### PR DESCRIPTION
**Pull request by jacknagel**

### CODE REVIEW

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    - name: Docker Login
+      uses: docker/login-action@v1.12.0
+      with:
+          registry: public.ecr.aws
+
+    - name: Docker Setup Buildx
+      uses: docker/setup-buildx-action@v1.6.0
+
+    - name: Build
```
Comment (jacknagel) [inline .github/workflows/docker-image.yml]:
```suggestion
    - name: Docker Build
```

Comment (mostafalarki1368mos) [reply .github/workflows/docker-image.yml]: []()_

[]()_

Comment (mostafalarki1368mos) [reply .github/workflows/docker-image.yml]: > []()_
> 
> []()_

- [ ]

Comment (mostafalarki1368mos) [reply .github/workflows/docker-image.yml]: > ```suggestion
>     - name: Docker Build
> ```

- [ ]

**Pull request merged**